lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -70,7 +71,7 @@
 '''
 
 RETURN = '''
-azure_vmss:
+vmss:
```

Comment:
User will get break if this field name is modifies.

Reply:
@yuwzho only the doc is modified, the module will return both formats till version 2.10